### PR TITLE
Refactor Member Index View Users Query 

### PR DESCRIPTION
**Pull request by juliannatetreault**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
This PR move eager loading from the Member Index View and into the `Admin::UsersController` by updating `Admin::UsersQuery` within the `index` method.

## Related Tickets & Documents
- Related Issue #16911 

## QA Instructions, Screenshots, Recordings
Ensure that all checks pass. ♻️ 

### UI accessibility concerns?
N/A

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: this is merely a refactor, so no underlying functionality has been altered.
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -31,6 +31,7 @@ class UsersController < Admin::ApplicationController
 
     def index
       @users = Admin::UsersQuery.call(
+        relation: User.includes([:organizations]),
```
Comment (aitchiss) [inline app/controllers/admin/users_controller.rb:34]: I noticed with this change we're overriding the default `UsersQuery` relation which is `User.registered`, and I can see invited users popping up in the main members index now - maybe just a small tweak needed there?

```suggestion
        relation: User.registered.includes([:organizations]),
```

**Review by aitchiss — COMMENTED** [review]
I'm seeing this bullet warning locally 🤔  

!["user: suzanne AVOID eager loading detected User => [:organization_memberships] Remove from your query: .includes([:organization_memberships])"](https://user-images.githubusercontent.com/20773163/162987798-d1d3e2c5-d441-48b8-947b-129f5a1f8c29.png)

File: app/controllers/admin/users_controller.rb (hunk 1 above)
Comment (juliannatetreault) [reply app/controllers/admin/users_controller.rb:34]: Well, that's unfortunate! I checked for this bullet warning on desktop and on mobile and it didn't show for me. 🙃 

Let me update this.

**Review by aitchiss — APPROVED** [review]
✨

**Review by Ridhwana — APPROVED** [review]
Nice one!

**Pull request merged**